src/views/FlawEditView.vue

```diff
@@ -41,9 +41,9 @@ function refreshFlaw() {
   <main>
     <FlawForm
       v-if="flaw"
+      :key="`${flaw.uuid}-${flaw.updated_dt}`"
```

Comment:
Why this change?
It's only the order of the props?

Reply:
Yes, is only the attribute order as it appeared to me on linting warns.